File: SelectionSort.java
class SelectionSort{

    public static void main(String[] args){

        int[] Nums = {38, 52, 9, 18, 6, 62, 13};
        int min, temp = 0;

        for(int i = 0; i < Nums.length; i++){

            min = i;
            for(int j = i+1; j < Nums.length; j++){

                if(Nums[j] < Nums[min]){
                    min = j;
                }
            }

            temp = Nums[i];
            Nums[i] = Nums[min];
            Nums[min] = temp;
        }

        for(int i = 0; i < Nums.length; i++){
            System.out.print(Nums[i] + " ");
        }
    }
}

File: selectionSort.js
const selectionSort = (Nums) => {

    for(let i = 0; i < Nums.length; i++){

        let minIndex = i;

        for(let j = i+1; j < Nums.length; j++){

            if(Nums[j] < Nums[minIndex]){
                minIndex = j;
            }
        }

        if(minIndex != i){
            [Nums[i], Nums[minIndex]] = [Nums[minIndex], Nums[i]];
        }
    }

    return Nums;
}

console.log(selectionSort([29, 10, 50, 37, 14]));